rules/cross-platform/command_and_control_non_standard_ssh_port.toml

```diff
@@ -0,0 +1,53 @@
+[metadata]
```

Comment:
We need to add a False positives section, with some guidance on reducing environment noise by adding exceptions.